.github/CONTRIBUTING.md

```diff
@@ -1,10 +1,11 @@
-# Contributing to ZFS on Linux
-<p align="center"><img src="http://zfsonlinux.org/images/zfs-linux.png"/></p>
+# Contributing to OpenZFS
+<p align="center">
+![img](https://openzfs.github.io/openzfs-docs/_static/img/logo/480px-Open-ZFS-Secondary-Logo-Colour-halfsize.png)
```

Comment:
I'm not sure this syntax is quite right.  It doesn't appear to be correctly showing the new image in the rich diff output.

Reply:
I copied the line directly from README.md but I guess there is some nuance in mixing HTML and markdown I wasn't aware of. Sticking to HTML here works :)